docs/src/model_library/cost_curves.md

```diff
@@ -0,0 +1,18 @@
+### Cost Curves
+```@docs
+CostCurve
+FuelCurve
+LinearCurve
```

Comment:
Since `CostCurve` and `FuelCurve` are composed of `LinearCurve`, `QuadraticCurve`, etc., it might be confusing to put them under the same heading.

Reply:
Resolved by https://github.com/NREL-Sienna/PowerSystems.jl/pull/1111/commits/212fb92018dc8cedb85f02c8654dcb9a4990a6ee